tools/python/util/mobile_helpers/test/test_check_model_can_use_ort_mobile_pkg.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import logging
+import onnx
+import pathlib
+import unittest
+
+from testfixtures import LogCapture
+from ..check_model_can_use_ort_mobile_pkg import run_check, run_check_with_model
+
+# example usage from <ort root>/tools/python
+# python -m unittest util/mobile_helpers/test/test_check_model_can_use_ort_mobile_pkg.py
+# NOTE: at least on Windows you must use that as the working directory for all the imports to be happy
+
+script_dir = pathlib.Path(__file__).parent
+ort_root = script_dir.parents[4]
+
+ort_package_build_config_filename = \
+    ort_root / 'tools' / 'ci_build' / 'github' / 'android' / 'mobile_package.required_operators.config'
+
+
+def _create_logger():
+    logger = logging.getLogger('default')
+    logger.setLevel(logging.DEBUG)
+    return logger
+
+
+class TestMobilePackageModelChecker(unittest.TestCase):
```

Comment:
>TestMobilePackageModelChecker [](http://example.com/codeflow?start=6&length=29)

nice, good to have some tests :)
are these run as part of a build?

Reply:
no - they're currently manual as running python tests of our helper scrips isn't setup in any CI. can add a backlog task to look at doing so.